src/components/farmModal.html

```diff
@@ -93,7 +93,7 @@
                                     </ul>
                                     <div class="justify-content-center py-1">
                                         <div class="btn-group btn-block">
-                                            <button type="button" class="btn btn-sm btn-secondary"
+                                            <button type="button" class="btn btn-sm btn-secondary mulchMultiplier"
```

Comment:
Standard bootstrap would be to use either `col-3` or `col-4` as class to set the width, instead of creating a new class, only for adding width